src/components/Message/utils.tsx

```diff
@@ -305,7 +305,7 @@ export const messageHasReactions = <
   StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics
 >(
   message?: StreamMessage<StreamChatGenerics>,
-) => !!message?.latest_reactions && !!message.latest_reactions.length;
+) => Object.values(message?.reaction_counts ?? {}).some((count) => count > 0);
```

Comment:
This check has never been strictly correct (we might have some reactions, but all of them may be unsupported) - but at least it's more in line with the behavior of the `ReactionsList` component.